searx/static/themes/oscar/js/searx.js

```diff
@@ -199,7 +202,175 @@ $(document).ready(function(){
         tabs.children().attr("aria-selected", "false");
         $(a.target).parent().attr("aria-selected", "true");
     });
+
+    /**
+     * Layout images according to their sizes
+     */
+    searx.image_thumbnail_layout = new searx.ImageLayout('#main_results', '#main_results .result-images', 'img.img-thumbnail', 200);
+    searx.image_thumbnail_layout.watch();
 });
+;/**
```

Comment:
I guess this is a code duplication of  [`searx/static/themes/oscar/src/js/imageresult.js`](https://github.com/searx/searx/pull/2675/files#diff-3051b1de9173fce6d86a7260a2462e9ab106608ae39f9148ca0bb9b043f037a3)?

Reply:
Yes 
https://github.com/searx/searx/blob/46db9a8600f1aec5c0780b0248fb6476c1ad5078/searx/static/themes/oscar/src/js/imageresult.js
is nearly duplicated of
https://github.com/searx/searx/blob/46db9a8600f1aec5c0780b0248fb6476c1ad5078/searx/static/themes/simple/js/searx_src/searx_imageresult.js

I'm going to keep only one file.

Reply:
Updated.

Now, using grunt, [`searx/static/themes/__common__/js/image_layout.js`](https://github.com/searx/searx/blob/b9e8ed063641d168e1a8318a5a07457f8e05e957/searx/static/themes/__common__/js/image_layout.js) is embedded in:
* `searx/static/themes/oscar/js/searx.min.js`
* `searx/static/themes/simple/js/searx.min.js`